edb/server/protocol/auth_ext/http.py

```diff
@@ -138,6 +138,14 @@ async def handle_request(
         )
 
         try:
+            try:
+                request.load_cookies()
+            except http.cookies.CookieError as ex:
+                raise errors.InvalidData(
+                    f'invalid cookies header: {ex}, '
+                    f'try clearing cookies and trying again'
+                )
```

Comment:
Seems better than the status quo, for sure! I wonder if we can do one better and actually ignore and clear out bad cookies based on the error? Conceptually, I always feel like cookies are things owned by the server, not but the client. If course, it's possible that the cookies are just completely mangled to the point where it's impossible to even know which key needs to be cleared.

Reply:
Maybe the best approach is to have some custom cookie parsing that just ignores invalid cookies (we can log a warning), as we only need to read the known good cookies we set anyway.

Reply:
The downside of ignoring invalid cookies is that you don't know it's messed up, right? I guess another approach is to lazily load cookies at the point when we need them and to do it selectively rather than populating a whole cookie jar, just ask for the morsels we need? Because then if something is messed up, there is clearly a bug. This isn't a blocker, though: this is still superior to the status quo in every way.